packages/contractkit/src/wrappers/MultiSig.ts

```diff
@@ -0,0 +1,34 @@
+import { TransactionObject } from 'web3/eth/types'
+import { Address } from '../base'
+import { MultiSig } from '../generated/types/MultiSig'
+import { BaseWrapper, proxyCall, stringToBytes, toTransactionObject } from './BaseWrapper'
+
+/**
+ * Contract for handling multisig actions
+ */
+export class MultiSigWrapper extends BaseWrapper<MultiSig> {
+  /**
+   * Allows an owner to submit and confirm a transaction.
+   * @param index The index of the pending withdrawal to withdraw.
+   */
+  async submitOrConfirmTransaction(destination: string, txObject: TransactionObject<any>) {
+    const data = stringToBytes(txObject.encodeABI())
+    const transactionCount = await this.contract.methods.getTransactionCount(true, true).call()
+    let transactionId
+    for (transactionId = 0; transactionId < Number(transactionCount); transactionId++) {
+      const transaction = await this.contract.methods.transactions(transactionId).call()
+      if (transaction.data === data && transaction.destination === destination) {
```

Comment:
nit: Consider `return toTransactionObject(this.kit, this.contract.methods.confirmTransaction(transactionId))` here instead of breaking. IMO would make the next few lines mildly more readable

Reply:
yes -- much cleaner